cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/LocalManager.java

```diff
@@ -91,14 +90,14 @@ public LocalManager(CConfiguration configuration, PipelineFactory pipelineFactor
     this.programTerminator = programTerminator;
     this.datasetFramework =
       new NamespacedDatasetFramework(datasetFramework,
-                                     new DefaultDatasetNamespace(configuration, Namespace.USER));
+                                     new DefaultDatasetNamespace(configuration, co.cask.cdap.data.Namespace.USER));
```

Comment:
same here. Whats wrong with importing data.Namespace? It doesn't conflict with Id.Namespace